tests/integration/test_model.py

```diff
@@ -334,3 +338,73 @@ def test_describe_model(c, training_df):
 
     with pytest.raises(RuntimeError):
         c.sql("DESCRIBE MODEL undefined_model")
+
+
+def test_export_model(c, training_df, tmpdir):
+    with pytest.raises(RuntimeError):
+        c.sql(
+            "EXPORT MODEL not_available_model with "
+            "(model_serde ='pickle',"
+            " location = '/tmp/model.pkl')"
+        )
+
+    c.sql(
+        f"""
+        CREATE MODEL IF NOT EXISTS my_model WITH (
+            model_class = 'sklearn.ensemble.GradientBoostingClassifier',
+            target_column = 'target'
+        ) AS (
+            SELECT x, y, x*y > 0 AS target
+            FROM timeseries
+            LIMIT 100
+        )
+    """
+    )
+    # Happy flow
+    temporary_file = tmpdir + "/pickle_model.pkl"
```

Comment:
You could use `os.path.join(tmpdir, "pickle_model.pkl")`, which works on operation systems (maybe need to import `os` before)

Reply:
makes sense 👍